runtime/pangolin/src/weights/pallet_bridge_grandpa.rs

```diff
@@ -74,17 +74,15 @@ impl<T: frame_system::Config> pallet_bridge_grandpa::WeightInfo for WeightInfo<T
 	/// Proof: BridgeMoonbaseGrandpa ImportedHashes (max_values: None, max_size: Some(36), added: 2511, mode: MaxEncodedLen)
 	/// The range of component `p` is `[51, 102]`.
 	/// The range of component `v` is `[50, 100]`.
-	fn submit_finality_proof(p: u32, v: u32, ) -> Weight {
+	fn submit_finality_proof(p: u32, _v: u32, ) -> Weight {
 		// Proof Size summary in bytes:
 		//  Measured:  `2544 + p * (40 ±0)`
 		//  Estimated: `165335`
-		// Minimum execution time: 1_610_178_000 picoseconds.
-		Weight::from_parts(53_319_117, 0)
+		// Minimum execution time: 1_790_557_000 picoseconds.
+		Weight::from_parts(2_881_401_060, 0)
```

Comment:
This value has increased dramatically. A bit weird.

Reply:
Not sure. You can run the benchmark to check again.

Reply:
I'll take a deeper look of this issue later.